### ISSUE TITLE
Running tests in subdirectory

### ISSUE THREAD

**Issue by janpaul123** [codecov/support#377]
Hi! At @remix we have a monorepo (remix/remix), with multiple directories that have their own test runners. Is there a way to specify the subdirectory/path when uploading coverage reports? Right now all files show up in Codecov as if they were in the root of the repo instead of in a subdirectory.

Combined with flags and hopefully soon https://github.com/codecov/support/issues/355 this would make monorepos work well in Codecov!

**Comment by stevepeak** [comment]
Hey @janpaul123 I believe I got you in Intercom, right? Did the targeting uploading work?

**Comment by janpaul123** [comment]
Yeah, that's possible, I don't quite remember. What do you mean by "targeting uploading" exactly? See https://codecov.io/gh/remix/remix/pull/22 for the PR in which I try to fix Codecov.
